tcl/windows/browser.tcl

```diff
@@ -56,10 +61,10 @@ proc ::gbrowser::new {base gnum {ply -1}} {
   bind $w <Right> "::gbrowser::update $n +1"
   bind $w <Up> "::gbrowser::update $n -10"
   bind $w <Down> "::gbrowser::update $n +10"
-  bind $w <minus> {set ::gbrowser::size [::board::resize %W.bd -1]}
```

Comment:
I think that something like:
`"set ::gbrowser::size \[::board::resize $w.bd -1 \]"`
should work and avoid adding another function that accepts a parameter `n`